platform/core/src/core.rs

```diff
@@ -117,7 +126,49 @@ pub fn load_feature_categories(
     )
 }
 
-pub fn exec_rdf_query(query: SPARQLQuery) -> Result<String, CoreFailure> {
+fn open_store(fs_root: String, active_feature_id: String) -> Result<Store, CoreFailure> {
+    Store::open(Path::new(&format!("{}/{}/{}", fs_root, active_feature_id, env!("RDF_DB_PATH")))).map_err(CoreFailure::map_storage_error)
```

Comment:
Maybe check the path elsewhere? A lot of stuff going on in a single line...

Reply:
Can add some returns, but after merge is done